src/app/code/community/Omise/Gateway/etc/system.xml

```diff
@@ -133,6 +133,27 @@
                             </fields>
                         </omise_offsite_internet_banking>
 
+                        <omise_offsite_citipoints type="group">
+                            <label>Pay With Citipoints</label>
```

Comment:
Should be "Citi points" - Citi themselves never write this as one word as far as I can tell

Reply:
https://omise-cdn.s3.amazonaws.com/assets/screenshots/Omise%20announces%20its%20partnership%20with%20Citibank%20Thailand/Omise%20__%20Citi%20Team_1.jpg

Reply:
or `Citi Pay with Points`

Reply:
"Pay with City Pay with Points" would sound totally ridiculous here - I was just advocating not writing Citi points as Citipoints to follow Citi's lead. So either "Pay with Citi Points" or "Citi Pay with Points" work (both can be seen elsewhere online) - although we should probably standardise with the latter one already in Omise literature 👍

Reply:
yes, I would change it to `Citi Pay with Points`